dm/master/scheduler/scheduler.go

```diff
@@ -326,6 +326,34 @@ func (s *Scheduler) AddSourceCfg(cfg *config.SourceConfig) error {
 	return nil
 }
 
+// UpdateSourceCfg update the upstream source config to the cluster.
```

Comment:
what should we do if there're already tasks using this source?

Reply:
i think we should return error here (just for now, maybe we can support dynamic change source config in this Sprint) 
fixed in https://github.com/pingcap/dm/pull/2083/commits/bc0ff0ac2d299d4dc10e0179a1347699b931a342